auth/auth.js

```diff
@@ -23,7 +25,7 @@ export async function hash(password) {
 }
 
 export function authenticateToken(req) {
-    const token = extractTokenFromHeader(req);
+    const token = auth.extractTokenFromHeader(req);
```

Comment:
Self-import and using indirection enables jest to mock extractTokenFromHeader from external file.